setParameters.m

```diff
@@ -34,22 +34,31 @@
 
     %% Experiment Design
 
+%     cfg.design.motionType = 'translation'; 
+%     cfg.design.motionType = 'radial';
+    cfg.design.motionType = 'translation';
     cfg.design.names = {'static'; 'motion'};
     cfg.design.nbRepetitions = 4;
     cfg.design.nbEventsPerBlock = 12; % DO NOT CHANGE
 
     %% Timing
     
+    % FOR 7T: if you want to create localizers on the fly, the following must be
```

Comment:
Our TR will be around 1.8 ms and we to have only one "condition" per volume so we have to make sure some of our parameters are a multiple of the TR.

Reply:
I think we should add an automatic check for this that send an error in case we don't the math wrong.